2/Dockerfile.rhel7

```diff
@@ -91,7 +91,7 @@ RUN rm /opt/openshift/base-plugins.txt && \
     /usr/local/bin/fix-permissions /opt/openshift && \
     chown -R 1001:0 /opt/openshift && \
     # the prior chown doesn't traverse the /opt/openshift/plugins links .. this one will assist fix-permission/assemble for extension builds like master/slave
-    chown 1001:0 /usr/lib64/jenkins/*hpi && \ 
+    chown 1001:0 /usr/lib/jenkins/*hpi && \ 
```

Comment:
seems like the new rpm installs all the plugins in a different location than the old rpms did?